cmd/controller-manager/app/controller_manager.go

```diff
@@ -143,6 +143,8 @@ func Run(controllerManagerOptions *options.ControllerManagerServer) error {
 		// TODO: disambiguate API errors
 		return fmt.Errorf("failed to get Service Catalog client configuration: %v", err)
 	}
+	// TODO (nilebox): Dirty temporary hack. Should whether accept as flag for application, or handle TLS correctly
+	serviceCatalogKubeconfig.Insecure = true
```

Comment:
This is probably the worst hack in this PR, other places at least disable TLS verification through config/flags.